### DISCUSSION TITLE
Is it ok to use hooks within a secondary builder?

### DISCUSSION THREAD

**Discussion by galacticgibbon** [rrousselGit/flutter_hooks#382] [General]
I was wondering if using hooks within a secondary builder like below causes any issues, being mindful of your list of do's and dont's?

The code seems to work! Just wanted to make sure :)

<img width="553" alt="Screenshot 2023-09-25 at 20 36 22" src="https://github.com/rrousselGit/flutter_hooks/assets/9589803/05ac28dc-3d68-48ca-88e5-94486ec551c8">

**Comment by rrousselGit** [comment]
That only works if that other builder is a HookWidget too.
